style.css

```diff
@@ -3,7 +3,7 @@
 body {
```

Comment:
- [ Optional ] Please  fix this weird looking border in the design it doesn't match the design 
![image](https://user-images.githubusercontent.com/74744787/214617308-5de2b7cf-2c62-4169-97d8-d5071e07ba98.png)

Reply:
Thank you and implemented